### PR TITLE
Start documentation changes for kubernetes

### PR DESCRIPTION
**Pull request by grantseltzer**
## Initial Checklist

- [ ] There is an issue describing the need for this PR.
- [x] Git log contains summary of the change.
- [ ] Git log contains motivation and context of the change.
- [ ] If part of an EPIC, PR git log contains EPIC number.
- [ ] If part of an EPIC, PR was added to EPIC description.

## Description (git log)

This is the start of changes to documentation in tracee with a particular focus
on changing the entry to be towards the user and less so the contributor. The
idea is also to push the use case of tracee for kubernetes and less on running
locally with docker.

Signed-off-by: grantseltzer <grantseltzer@gmail.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -122,6 +130,21 @@ $ docker run \
...
+1. Install tracee via helm.
+
+See the kubernetes [installation guide].
+
+2. FIXME:
+
+- How are users interacting with Tracee via Kubernetes?
+  - Are signature alerts printed to stderr and then collected via logs?
+  - Is Postee enabled by default and if so how is it configured?
+  - How are signatures configured?
```
Comment (grantseltzer) [inline docs/index.md:146]: @josedonizetti @simar7 Perhaps you two can help me with this?

File: docs/installing/kubernetes.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ the detections in your preferred way (e.g. Slack, E-mail, JIRA and more).
 
         ```text
         $ helm repo add aqua-charts https://aquasecurity.github.io/helm-charts
-        $ helm dependency update ./deploy/helm/tracee
-        $ helm install tracee ./deploy/helm/tracee \
+        $ helm dependency update
+        $ helm install tracee aqua/tracee \
```
Comment (grantseltzer) [inline docs/installing/kubernetes.md:33]: @josedonizetti Can you confirm this is correct (for once #2256 merges)?

**Comment by grantseltzer** [comment]
To discuss:

- Would it be helpful to have a tutorial of installing tracee to a minikube cluster? or something similar?
- There's a video in the README, it would be great if we can record a new demo at the beginning of Kubecon to replace this.

File: docs/index.md (hunk 1 above)
Comment (simar7) [reply docs/index.md:146]: sorry don't have a context but I'll share what I know about tracee + k8s:
- tracee is deployed via a helm chart https://github.com/aquasecurity/tracee/blob/main/deploy/helm/tracee/values.yaml
- this helm chart has an included dependency on postee, which is how today people can see their tracee rules events showing up in the form of an UI.
- there's nothing special in terms of configuring signatures, the default ones are loaded to be run.

Comment (josedonizetti) [reply docs/index.md:146]: @grantseltzer @simar7 I think we should make postee not enabled by default. The first interaction of a user with tracee would be through stdout logs. Than using postee+tracee should be a tutorial, where we can dive more into postee usage. 

Yes, currently all signatures will be loaded by default, and there isn't a way to configure it.

File: docs/installing/kubernetes.md (hunk 2 above)
Comment (josedonizetti) [reply docs/installing/kubernetes.md:33]:
```
helm repo add aqua https://aquasecurity.github.io/helm-charts/
helm repo update
helm dependency update aqua/tracee
helm install tracee aqua/tracee \
  --namespace tracee-system \
  --create-namespace
```

**Comment by josedonizetti** [comment]
1+ to have a getting started with minikube, or kind

**Comment by rafaeldtinoco** [comment]
> 1+ to have a tutorial with minikube, or kind

Be aware that kind needs to exclude other tracee-ebpf processes from the tracing (as all nodes are containers and not vms or real nodes).

**Comment by grantseltzer** [comment]
Closing in favor of #2272

**Pull request closed without merging**